### ISSUE TITLE
请问下蒸馏loss这儿的问题

### ISSUE THREAD

**Issue by gangxu822** [XLearning-SCU/2024-ICML-TAC#5]
![image](https://github.com/user-attachments/assets/820b31bc-13bb-4da1-954c-15b3ca64ba9e)
![image](https://github.com/user-attachments/assets/b1f558bc-0431-4c7f-84a2-0e954a15b297)

为啥这儿labels全是0

**Comment by Yunfan-Li** [comment]
这里把正样本对的相似性全部放在首位了，所以每一行对应 cross entropy 的 label 均为 0

**Comment by gangxu822** [comment]
> 这里把正样本对的相似性全部放在首位了，所以每一行对应 cross entropy 的 label 均为 0

是不是可以认为这儿只是占位的，全部设成1也行？跟全部设成1一样的效果

**Comment by Yunfan-Li** [comment]
不是

**Comment by gangxu822** [comment]
![image](https://github.com/user-attachments/assets/8a8639bb-3785-48df-89e5-9b4a182388fb)
这儿logits第一列是positive_clusters ，后面列是negative_clusters把？

**Comment by Yunfan-Li** [comment]
是的

**Comment by gangxu822** [comment]
> 不是

0代表下标把，代表第一个类，因为把postive_cluster放在第一列，所以类标号就是0？

**Comment by Yunfan-Li** [comment]
对的

**Comment by mzdxx** [comment]
> 对的

想请问一下作者，论文中表示软分配矩阵的第i行代表的是一个样本软聚类分配概率，第i列应该代表的是一个簇的整体结构特性——即哪些样本倾向于归属于这个簇。这里的蒸馏损失使用是软分配矩阵的第i列，那么这个蒸馏损失是不是在鼓励不同模态视角下的聚类簇展现出结构上的一致性？

**Comment by Yunfan-Li** [comment]
是的，蒸馏损失是在跨模态邻域之间进行的，可以认为是鼓励不同模态视角下的聚类簇展现出结构上的一致性

**Comment by mzdxx** [comment]
> 是的，蒸馏损失是在跨模态邻域之间进行的，可以认为是鼓励不同模态视角下的聚类簇展现出结构上的一致性

好的，多谢作者的回复
